pkg/registry/servicecatalog/instance/storage.go

```diff
@@ -0,0 +1,44 @@
+package instance
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/rest"
+	genericregistry "k8s.io/kubernetes/pkg/registry/generic"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+type serviceInstanceStorage struct {
+}
+
+// NewStorage creates a new rest.Storage responsible for accessing Instance
+// resources
+func NewStorage() rest.Storage {
```

Comment:
I've got something like this, as well. Do we really have to have one of these for each type we are storing? Do we really need to write a *`List` type?